src/pdn/src/PdnGen.tcl

```diff
@@ -3408,11 +3408,13 @@ proc generate_lower_metal_followpin_rails {} {
       set vdd_name [get_voltage_domain_power [get_voltage_domain $xMin [expr $vdd_y - $width / 2] $xMax [expr $vdd_y + $width / 2]]]
       set vss_box [::odb::newSetFromRect $xMin [expr $vss_y - $width / 2] $xMax [expr $vss_y + $width / 2]]
       set vss_name [get_voltage_domain_ground [get_voltage_domain $xMin [expr $vss_y - $width / 2] $xMax [expr $vss_y + $width / 2]]]
+      # generate power_rails using first domain_power
+      set first_power_name [lindex $vdd_name 0]
       # debug "[$box xMin] [expr $vdd_y - $width / 2] [$box xMax] [expr $vdd_y + $width / 2]"
-      if {$vdd_name == [get_voltage_domain_power [dict get $design_data core_domain]]} {
+      if {$first_power_name == [get_voltage_domain_power [dict get $design_data core_domain]]} {
         add_stripe $lay "POWER" $vdd_box
```

Comment:
I suggest using primary_power of a specific domain for STD cell Power net only, rather than giving a list and selecting the first net. Additional nets can be added to a separate variable.

Reply:
Yes. I'll try to create `additional_power ` variable.

Reply:
maybe it should be called `secondary_power `?